### DISCUSSION TITLE
Separation of Nodes manager, with an Addon Manager

### DISCUSSION THREAD

**Discussion by BrechtCorbeel** [comfyanonymous/ComfyUI#4354] [Ideas]
Not only do I find the naming confusing, I tend to actively forget and it is not intuitive that the Node Manager also includes addons that have nothing to do with nodes.

Like for example, I finally found the queue saver, but it was in the Nodes Manager menu, which is not where I would expect something to do with comfy's GUI functionality.

This would also be great so that you could get Theme's or whatever optimisations there, well defined in a proper category.

**Comment by ltdrdata** [comment]
What does the Queue Saver you're referring to mean?

**Reply by BrechtCorbeel** [reply]
> What does the Queue Saver you're referring to mean?

well it's a javascript addon that allows you to save the queue, which is not a node but a functionality present in the GUI when launching Comfy.

You can find it in the nodes manager.

**Reply by ltdrdata** [reply]
Are you saying that an extension called `ComfyUI-SaveQueues` can be installed from the Node Manager?

**Reply by BrechtCorbeel** [reply]
> Are you saying that an extension called `ComfyUI-SaveQueues` can be installed from the Node Manager?

Yes, I found it before and was going to implement it but then I though why not take a look there and it is not the only extension that is in the nodes manager there is a bunch of them.

Which is why I am pointing out it is bizarre that extensions that have nothing to do with nodes aren't a category of their own.

**Reply by BrechtCorbeel** [reply]
> Are you saying that an extension called `ComfyUI-SaveQueues` can be installed from the Node Manager?

My point is if for example you use Blender 3D you don't go looking for your extensions in your 3d nodes categories you go to the tab for extensions, but there is no tab for extensions so they are lumped in with the nodes.

**Reply by ltdrdata** [reply]
Currently, there is no system in place to categorize in this way.
To address this issue, discussions will be necessary regarding both technical aspects and policy considerations.

Furthermore, since ComfyUI-Manager is not yet an integrated part of ComfyUI, this matter is not likely to be addressed immediately.
